marker_rustc_driver/src/bin/marker_rustc_driver.rs

```diff
@@ -0,0 +1,68 @@
+#![doc = include_str!("../../README.md")]
```

Comment:
Today I learned that you can have multiple binaries simply by adding a `bin` folder under source o.O

Cargo's documentation states:

> * The default executable file is `src/main.rs`.
>   * Other executables can be placed in `src/bin/`.

Could you move this file to `src/main.rs`? For me, that's the default location where I would look first, and it's also the default executable, which should be located in `src/main.rs` according to the docs.

I assume, this was just a stylistic choice, as moving it to `src/main.rs` seems to work the same way. If there is another reason, please share it :)

Reply:
> I assume, this was just a stylistic choice

To be honest, I didn't know a package could have both a `src/lib.rs` and a `src/main.rs`. So, TIL too. :)